test/Prelude.purs

```diff
@@ -98,15 +98,20 @@ shouldEqualStr
   -> String
   -> m Unit
 shouldEqualStr v1 v2 =
+  let
+    renderNonPrinting =
+      String.replaceAll (String.Pattern "\r") (String.Replacement "␍")
+      >>> String.replaceAll (String.Pattern "\t") (String.Replacement "␉-->")
+  in
```

Comment:
If this is contentious i can undo, just thought it would be a good unobtrusive addition to catch failures early in CI

Reply:
Ah yeah, I like it! (for now, let's see how it fares over time 😂)